OutOfSchool/OutOfSchool.AuthCommon/Validators/CustomPasswordValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Extensions.Localization;
+
+namespace OutOfSchool.AuthCommon.Validators;
+
+public class CustomPasswordValidator : IPasswordValidator<User>
+{
+    private readonly IStringLocalizer<SharedResource> localizer;
+    private readonly ICustomPasswordRules passwordRules;
+
+    public CustomPasswordValidator(IStringLocalizer<SharedResource> localizer)
+    {
+        this.localizer = localizer;
+        passwordRules = new CustomPasswordRules();
+    }
```

Comment:
(Code style) Could this constructor be removed? 

Asking, because manually created class instances (like `new CustomPasswordRules()` we have here) is usually a code smell, that create high coupling and break testability. It's still possible for edge cases, though not for current one (at least I believe so).

Reply:
Fixed. Thank you!